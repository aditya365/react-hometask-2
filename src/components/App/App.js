import React, { Component } from "react";

// export default class App extends Component {
//   render() {
//     return (
//       <div>
//         <h1>App Component</h1>
//       </div>
//     );
//   }
// }

const App = () => {
  return <h1>App Component!!</h1>;
};


export default App;