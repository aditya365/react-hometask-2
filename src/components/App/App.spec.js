import React from "react";
import renderer from "react-test-renderer";

import App from "./App";

test("renders learn react link", () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
