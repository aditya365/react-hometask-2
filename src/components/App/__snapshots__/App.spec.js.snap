// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component snapshot renders 1`] = `"<App/>"`;

exports[`renders learn react link 1`] = `
<h1>
  App Component!!
</h1>
`;
